### PR TITLE
Add contact me section

### PR DESCRIPTION
**Pull request by PedroH-Neves**
- Finished the contact me section
- Add all requirements for contact section
- Add more css styles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -216,10 +229,10 @@ <h3 class="about-title secundary-color">About Me</h3>
...
         in touch.</h3>
-      <form action="/" class="contact-form">
-        <input id="name" class="form-input" type="text" placeholder="Full name">
-        <input id="email" class="form-input" type="email" placeholder="Email address">
-        <textarea class="text-area placeholder" placeholder="Write here"></textarea>
+      <form action="https://formspree.io/f/mgeqvaev" method="post" class="contact-form">
+        <input id="name" name="clientName" class="form-input" type="text" placeholder="Full name" maxlength="30" required>
+        <input id="email" name="clientEmail" class="form-input" type="email" placeholder="Email address" required>
+        <textarea class="text-area placeholder" placeholder="Write here" maxlength="500" required></textarea>
         <button class="submit-btn primary-color button-hover button-pressed" type="button">Get in touch</button>
```
Comment (phelian23) [inline index.html]: - [x] Kindly make use of `type=" submit"` for this button, as it is used to submit the form

File: style.css [hunk 2]
```diff
@@ -293,8 +293,9 @@
 }
 
 .form-input {
-  border-style: solid;
+  border-style: none;
   border-bottom: 1px solid #c2c7cf;
+  width: 45%;
   padding: 10px 0 10px 20px;
   color: #979493;
 }
```
Comment (phelian23) [inline style.css]: - [x] Kindly ensure that your design matches the Figma template as much as possible. Please take a look at the width of the form fields.
Your design
Capture
Figma template
figma

- [x] Also kindly align the text above the form input to the center

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Team, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Great job setting up the correct linters ✔️
Professional documentation ✔️
Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -216,11 +229,11 @@ <h3 class="about-title secundary-color">About Me</h3>
...
-        <input id="name" class="form-input" type="text" placeholder="Full name">
-        <input id="email" class="form-input" type="email" placeholder="Email address">
-        <textarea class="text-area placeholder" placeholder="Write here"></textarea>
-        <button class="submit-btn primary-color button-hover button-pressed" type="button">Get in touch</button>
+      <form action="https://formspree.io/f/mgeqvaev" method="post" class="contact-form">
+        <input id="name" name="clientName" class="form-input" type="text" placeholder="Full name" maxlength="30" required>
+        <input id="email" name="clientEmail" class="form-input" type="email" placeholder="Email address" required>
+        <textarea class="text-area placeholder" placeholder="Write here" maxlength="500" required></textarea>
+        <button class="submit-btn primary-color button-hover button-pressed" type="submit">Get in touch</button>
       </form>
```
Comment (ichala) [inline index.html:237]: - [ ] as the previous reviewer requested kindly fix the margins and the alignments to match Figma design

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @PedroH-Neves , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Correct workflow ✅ 
-  Form submission and validation are perfect ✅ 

 ## Required Changes ♻️

- [ ] **_Check the comments below_**

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @PedroH-Neves  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Correct Workflow ✔️
- Changes Implemented ✔️

 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- N/A



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**